Controller/GraphQLController.php

```diff
@@ -132,4 +123,63 @@ private function getPayload()
 
         return [$queries, $isMultiQueryRequest];
     }
+
+    private function initializeSchemaService()
+    {
+        if ($this->container->initialized('graphql.schema')) {
+            return;
+        }
+
+        $this->container->set('graphql.schema', $this->makeSchemaService());
+    }
+
+    private function makeSchemaService()
+    {
+        if ($this->container->has($this->getSchemaService())) {
```

Comment:
here if $this->getSchemaService returns `null` because the schemaService is not defined, it can create bugs within Symfony trying to get the "null" service